x-pack/platform/plugins/private/reporting/public/management/components/create_scheduled_report_form.tsx

```diff
@@ -99,7 +103,17 @@ export const CreateScheduledReportForm = ({
           ...(reportTypeId === 'printablePdfV2' ? { optimizedForPrinting } : {}),
         }),
         schedule: { rrule: rrule as Rrule },
-        notification: sendByEmail ? { email: { to: emailRecipients } } : undefined,
+        notification: sendByEmail
+          ? {
+              email: {
+                to: emailRecipients,
+                cc: emailCcRecipients,
```

Comment:
The cc and bcc fields were already supported by the backend but not on the FE, added these too